### ISSUE TITLE
Build is failing

### ISSUE THREAD

**Issue by wavebeem** [jneen/parsimmon#45]
The [build is failing](https://travis-ci.org/jneen/parsimmon/jobs/67448663) with this message:

```
npm ERR! Error: No compatible version found: camelcase@'^1.0.2'
```

This happened on the *Adding sepBy and sepBy1* commit, and it appears to have resulted in those functions not being available in the npm package.

**Comment by anko** [comment]
Works locally, so I think the package contents are fine.

[Sounds like](https://github.com/npm/npm/issues/4984) that Travis instance had an old `npm`. A rebuild (![shot](https://cloud.githubusercontent.com/assets/5231746/9628562/30ae0b62-516c-11e5-9c87-65fa3f980ce3.png) in Travis UI) might be enough.

**Comment by anko** [comment]
I think this issue can be safely closed now: #55 did it. I'd have [closed by commit](https://help.github.com/articles/closing-issues-via-commit-messages/), but wasn't sure at that point yet whether my changes would fix it.

**Comment by jneen** [comment]
yep!
